packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Added
+  * Added documentation updates for `Single Select` in `terra-form-select`.
```

Comment:
Should this entry be here?

Reply:
It should not, good catch, too many conflict resolves in changelogs
fixed here [3d675a5](https://github.com/cerner/terra-core/pull/3840/commits/3d675a58f91d1405e29996f78d4a6b876665654d)